modules/plugin/mongodb/src/test/java/org/geotools/data/mongodb/MockHTTPClient.java

```diff
@@ -0,0 +1,102 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geotools.data.mongodb;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import org.geotools.data.ows.HTTPClient;
+import org.geotools.data.ows.HTTPResponse;
+
+/** @author ImranR */
+public class MockHTTPClient implements HTTPClient {
+
+    /** Mocks HTTP calls */
+    Map<String, HTTPResponse> expectGet = new HashMap<String, HTTPResponse>();
+
+    public MockHTTPClient() {}
+
+    public MockHTTPClient(Map<String, HTTPResponse> expectGet) {
+        this.expectGet = expectGet;
+    }
+
+    public void expectGet(String url, HTTPResponse response) {
+        expectGet.put(url, response);
+    }
+
+    @Override
+    public HTTPResponse post(URL url, InputStream postContent, String postContentType)
+            throws IOException {
+        // TODO Auto-generated method stub
```

Comment:
Remove all the autogenerated TODO comments in this file.

Reply:
will do